pkg/scale/scale.go

```diff
@@ -137,6 +137,7 @@ func (cc *cruiseControlScaler) Status(ctx context.Context) (StatusTaskResult, er
 			State:              v1beta1.CruiseControlTaskCompleted,
 		},
 		Status: &status,
+		Result: resp.Result,
```

Comment:
We should rename the Result -> State

Reply:
Addressed in adac7932a9524bfd57b4949693c7c0e558c1a247.